tests/driver_sx1280/main.c

```diff
@@ -0,0 +1,380 @@
+{
+    printf("Usage: %s set <freq|bw|sf|cr|> <value>\n", cmd);
+}
+
+static int sx1280_set_cmd(netdev_t *netdev, int argc, char **argv)
+{
+    if (argc == 3) {
+        if (!strcmp("freq", argv[2])) {
+            _usage_freq();
+        }
+        if (!strcmp("bw", argv[2])) {
+            _usage_bw();
+        }
+        if (!strcmp("sf", argv[2])) {
+            _usage_sf();
+        }
+        if (!strcmp("cr", argv[2])) {
+            _usage_cr();
+        }
+    }
+    if (argc != 4) {
+        _set_usage(argv[0]);
+        return -1;
+    }
+
+    int ret = 0;
+
+    if (!strcmp("freq", argv[2])) {
+        uint32_t freq = strtoul(argv[3], NULL, 10);
+        ret = netdev->driver->set(netdev, NETOPT_CHANNEL_FREQUENCY, &freq, sizeof(uint32_t));
+    }
+    else if (!strcmp("bw", argv[2])) {
+        uint16_t bw = atoi(argv[3]);
+        ret = netdev->driver->set(netdev, NETOPT_BANDWIDTH, &bw, sizeof(uint16_t));
+    }
+    else if (!strcmp("sf", argv[2])) {
+        uint8_t sf = atoi(argv[3]);
+        ret = netdev->driver->set(netdev, NETOPT_SPREADING_FACTOR, &sf, sizeof(uint8_t));
+    }
+    else if (!strcmp("cr", argv[2])) {
+        uint8_t cr = atoi(argv[3]);
+        ret = netdev->driver->set(netdev, NETOPT_CODING_RATE, &cr, sizeof(uint8_t));
+    }
+    else {
+        _set_usage(argv[0]);
+        return -1;
+    }
+
+    if (ret < 0) {
+        printf("cannot set %s\n", argv[2]);
+        return ret;
+    }
+
+    printf("%s set\n", argv[2]);
+    return 0;
+}
+
+static void _rx_usage(const char *cmd)
+{
+    printf("Usage: %s rx <start|stop>\n", cmd);
+}
+
+static int sx1280_rx_cmd(netdev_t *netdev, int argc, char **argv)
+{
+    if (argc == 2) {
+        _rx_usage(argv[0]);
+        return -1;
+    }
+
+    if (!strcmp("start", argv[2])) {
+        /* Switch to RX (IDLE) state */
+        netopt_state_t state = NETOPT_STATE_IDLE;
+        netdev->driver->set(netdev, NETOPT_STATE, &state, sizeof(state));
+        printf("Listen mode started\n");
+    }
+    else if (!strcmp("stop", argv[2])) {
+        /* Switch to STANDBY state */
+        netopt_state_t state = NETOPT_STATE_STANDBY;
+        netdev->driver->set(netdev, NETOPT_STATE, &state, sizeof(state));
+        printf("Listen mode stopped\n");
+    }
+    else {
+        _rx_usage(argv[0]);
+        return -1;
+    }
+
+    return 0;
+}
+
+static int sx1280_tx_cmd(netdev_t *netdev, int argc, char **argv)
+{
+    if (argc == 2) {
+        printf("Usage: %s tx <payload>\n", argv[0]);
+        return -1;
+    }
+
+    printf("sending \"%s\" payload (%u bytes)\n",
+           argv[2], (unsigned)strlen(argv[2]) + 1);
+    iolist_t iolist = {
+        .iol_base = argv[2],
+        .iol_len = (strlen(argv[2]) + 1)
+    };
+
+    if (netdev->driver->send(netdev, &iolist) == -ENOTSUP) {
+        puts("Cannot send: radio is still transmitting");
+        return -1;
+    }
+
+    return 0;
+}
+
+static int sx1280_reset_cmd(netdev_t *netdev, int argc, char **argv){
```

Comment:
```suggestion
static int sx1280_reset_cmd(netdev_t *netdev, int argc, char **argv)
{
```
Can you uncrustify again?